README.md

```diff
@@ -22,9 +22,8 @@ Problems and Configurations
 
   - Pass -DCMAKE_INSTALL_PREFIX to set the install location.
 
-  - Python and C++ bindings are built. Just make sure the relevant tools
-    are on your PATH.  Java and MATLAB bindings are now in the **archive** directory
-    and no longer built.
+  - Java, MATLAB, Python, C++ bindings all build. Just make sure the relevant tools
```

Comment:
Looks like a regression.

Reply:
Nope, that's intentional; the Java and MATLAB bindings "haven't been touched in years."

Reply:
I agree, but the change doesn't.

The original (current) text says Java and Matlab are "removed / archived", but the new text says they're all fine.